Nabia/HW4/debug.md

```diff
@@ -0,0 +1,11 @@
+# HW4 - Debug Report
```

Comment:
Looks good but you forgot to make the PR in a way that closes the issue number assigned to you. Please revisit the last bullet point of HW4